.github/workflows/ci-unit-tests-report.yml

```diff
@@ -2,13 +2,13 @@ name: Unit Test Results
 
 on:
   workflow_run:
-    workflows: ["Unit Tests"]
+    workflows: ["Unit Tests", "Unit Tests on Go Tip"]
     types:
       - completed
 
 jobs:
   unit-test-results:
-    name: Unit Test Results
+    name: ${{ github.event.workflow_run.name }}
```

Comment:
Ensures the title of the summary report is "Unit Tests on Go Tip summary" instead of just "Unit Tests summary"